hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/utils/db/cache/TestTableCache.java

```diff
@@ -78,10 +66,11 @@ public void create() {
     LOG.info("cacheType: {}", cacheType);
```

Comment:
I think log is no longer necessary, since parameterized runs are better labeled with JUnit5.  (Then `LOG` can be removed as well.)